Falanx.Ast/QuotationToAst.fs

```diff
@@ -62,8 +62,8 @@ type Quotations() =
                 //dependencies.Append v.Type
                 let vType = sysTypeToSynType range v.Type knownNamespaces ommitEnclosingType
                 let spat = SynSimplePat.Id(mkIdent range v.Name, None, false ,false ,false, range)
-                let untypedPat = SynSimplePats.SimplePats([spat], range)
-                let typedPat = SynSimplePats.Typed(untypedPat, vType, range)
+                //let untypedPat = SynSimplePats.SimplePats([spat], range)
```

Comment:
Let's don't leave commented out code. Remove it instead.

Reply:
I need it to retype the expression, Im going to retype all the things I untyped, then make a type stripper.

Reply:
Or maybe thats a bit a tweaked earlier on, i pretty certain some of the early mv stuff caused these later bugs